Gems/ImGui/Code/Include/ImGuiBus.h

```diff
@@ -99,6 +100,17 @@ namespace ImGui
     };
     using ImGuiManagerBus = AZ::EBus<IImGuiManager, IImGuiManagerRequests>;
 
+    class IImGuiManagerNotifications : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
+        using Bus = AZ::EBus<IImGuiManagerNotifications>;
+
+        virtual void ImGuiSetEnabled( [[maybe_unused]] bool enabled) {}
+    };
+    using ImGuiManagerNotificationBus = AZ::EBus<IImGuiManagerNotifications>;
+
```

Comment:
This could be made an AZ::Event<bool> that is stored on the IImGuiManagerRequests interface.
````
using ImGuiSetEnabledEvent = AZ::Event<bool>;
ImGuiSetEnabledEvent m_setEnabledEvent;
```

Reply:
Changes made and pushed.